src/component/Modal.tsx

```diff
@@ -111,8 +110,8 @@ const scrollableContainerStyles = (scrollbarWidth: number) => css`
     background-color: ${palette.neutral[100]};
     margin-right: -${scrollbarWidth}px;
     overflow: hidden;
-    max-height: 100%; /* For scrolling purposes */
-    height: 450px; /* For scrolling purposes */
+    display: flex;
+    flex-direction: column;
```

Comment:
@GHaberis Just checking if this has been tested on other browsers like IE11?

Reply:
Yep I've tested on IE 11 too 👍